solutions/my_rotate_solution.rb

```diff
@@ -1,7 +1,13 @@
 class Array
+  # define a method `my_rotate(positions)` that rotates the 
```

Comment:
Should be `Array#my_rotate` or otherwise specify that it's monkey patched onto the Array class